chat-components/src/components/outofofficehourspane/common/presetFourProps/presetFourOOOHPaneControlProps.ts

```diff
@@ -0,0 +1,9 @@
+import { IOOOHPaneControlProps } from "../../interfaces/IOOOHPaneControlProps";
+import { defaultOOOHPaneControlProps } from "../defaultProps/defaultOOOHPaneControlProps";
+
+export const presetFourOOOHPaneControlProps: IOOOHPaneControlProps = {
+    ...defaultOOOHPaneControlProps,
+    id: "oc-lcw-outofofficehourspane-preset1",
+    titleText: "We are out of office. Please contact us here https://www.microsoft.com for further assisstance",
```

Comment:
have we considerate to localize default messages?

Reply:
Unsure on this as this is just a dummy story for VRT displaying and validating this capability